John_Doe/css/styles.css

```diff
@@ -1,81 +0,0 @@
-html,
```

Comment:
Please don`t delete initial folder.
This pull request will replace initial folder "John_Doe" with "Anna_Golovenko".